src/usdb_syncer/usdb_scraper.py

```diff
@@ -410,9 +410,11 @@ def get_notes(song_id: SongId, logger: Log) -> str:
         payload={"wd": "1"},
     )
     soup = BeautifulSoup(html, "lxml")
-    text = _parse_song_txt_from_txt_page(soup)
-    return text
+    text, success = _parse_song_txt_from_txt_page(soup)
+    return text, success
 
 
-def _parse_song_txt_from_txt_page(soup: BeautifulSoup) -> str:
-    return soup.find("textarea").string  # type: ignore
+def _parse_song_txt_from_txt_page(soup: BeautifulSoup) -> tuple[str, bool]:
+    if textarea := soup.find("textarea"):
+        return textarea.string, True  # type: ignore
+    return "", False
```

Comment:
Can't we return `None` or an empty string in the error case here? I find that more readable and ergonomic than types you have to unpack first.

Reply:
Always appreciate your improvement suggestions!